### DISCUSSION TITLE
How to add animation of " Fade out (previous view) and  Slide in (new view) when the view page change ?

### DISCUSSION THREAD

**Discussion by segaduck** [AvaloniaUI/Avalonia#14363] [General]
I've check the document in official website and surf on the Internet for 2 days but still cannot make it work.
Anyone can help ?  Or Is there any sample code for me to study by myself ?
Thank you guys.

**Comment by timunie** [comment]
https://github.com/AvaloniaUI/Avalonia.Samples/tree/main/src/Avalonia.Samples/Routing/BasicViewLocatorSample

**Comment by thevortexcloud** [comment]
https://docs.avaloniaui.net/docs/guides/graphics-and-animation/page-transitions/page-transition-combinations
https://docs.avaloniaui.net/docs/reference/controls/detailed-reference/transitioningcontentcontrol
